ios/EscPosPrinter.m

```diff
@@ -220,11 +220,12 @@ + (BOOL)requiresMainQueueSetup
 
 
 RCT_EXPORT_METHOD(printBuffer: (NSArray *)printBuffer
+                  timeout:(int)timeout
```

Comment:
I think it'll be better if we could pass an object here like `params` in discovery. And I suppose it should be optional.

Reply:
will do